docs/security/authorization.md

```diff
@@ -8,38 +8,38 @@
-](https://cwiki.apache.org/confluence/display/RANGER/Ranger+Plugin+for+Hive+MetaStore). [Apache Ranger](https://ranger.apache.org/) makes the scope of existing SQL-Standard Based Authorization expanded but without supporting Spark SQL. [spark-authorizer](https://github.com/yaooqinn/spark-authorizer) sticks them together.
+[Apache Ranger](https://ranger.apache.org/)  is a framework to enable, monitor and manage comprehensive data security across the Hadoop platform but end before Spark or Spark SQL. The [spark-authorizer](https://github.com/NetEase/spark-authorizer) enables Kyuubi with control access ability reusing [Ranger Plugin for Hive MetaStore
+](https://cwiki.apache.org/confluence/display/RANGER/Ranger+Plugin+for+Hive+MetaStore). [Apache Ranger](https://ranger.apache.org/) makes the scope of existing SQL-Standard Based Authorization expanded but without supporting Spark SQL. [spark-authorizer](https://github.com/NetEase/spark-authorizer) sticks them together.
 
-Please refer to the [Spark SQL Ranger Security Support Guide](https://yaooqinn.github.io/spark-authorizer/docs/ranger_authorization.html) in the online documentation for an overview on how to configure Ranger for Spark SQL.
+Please refer to the [Spark SQL Ranger Security Support Guide](https://NetEase.github.io/spark-authorizer/docs/ranger_authorization.html) in the online documentation for an overview on how to configure Ranger for Spark SQL.
 
 ## Additional Documentations
 
-[Building Kyuubi](https://yaooqinn.github.io/kyuubi/docs/building.html)  
-[Kyuubi Deployment Guide](https://yaooqinn.github.io/kyuubi/docs/deploy.html)    
-[Kyuubi Containerization Guide](https://yaooqinn.github.io/kyuubi/docs/containerization.html)   
-[High Availability Guide](https://yaooqinn.github.io/kyuubi/docs/high_availability_guide.html)  
-[Configuration Guide](https://yaooqinn.github.io/kyuubi/docs/configurations.html)  
-[Authentication/Security Guide](https://yaooqinn.github.io/kyuubi/docs/authentication.html)  
-[Kyuubi Architecture](https://yaooqinn.github.io/kyuubi/docs/architecture.html)  
-[Home Page](https://yaooqinn.github.io/kyuubi/)
+[Building Kyuubi](https://NetEase.github.io/kyuubi/docs/building.html)  
+[Kyuubi Deployment Guide](https://NetEase.github.io/kyuubi/docs/deploy.html)    
+[Kyuubi Containerization Guide](https://NetEase.github.io/kyuubi/docs/containerization.html)   
+[High Availability Guide](https://NetEase.github.io/kyuubi/docs/high_availability_guide.html)  
+[Configuration Guide](https://NetEase.github.io/kyuubi/docs/configurations.html)  
+[Authentication/Security Guide](https://NetEase.github.io/kyuubi/docs/authentication.html)  
+[Kyuubi Architecture](https://NetEase.github.io/kyuubi/docs/architecture.html)  
+[Home Page](https://NetEase.github.io/kyuubi/)
```

Comment:
clear this part, these links are broken

Reply:
I removed `Hit Count` and `DepShield Badge`, it should be ok now.